### PR TITLE
Add 2 missing pkgs

### PR DESCRIPTION
**Pull request by ocefpaf**
@abkfenris do you have anything else we should add here? Trying to avoid creating yet another image :-)

### CODE REVIEW

File: environment.yml [hunk 1]
```diff
@@ -30,7 +31,6 @@ dependencies:
   - ioos_qc
   - ipyleaflet
   - jupyter
-  - jupyter-rsession-proxy
```
Comment (abkfenris) [inline environment.yml:33]: We need to keep this now.

Comment (ocefpaf) [reply environment.yml:33]: Argh. I forgot about that.

**Comment by abkfenris** [comment]
Those were the only packages that have popped out at me so far.

**Comment by ocefpaf** [comment]
Merging so people can run the argo notebook.

**Pull request merged**